commands_data.go

```diff
@@ -0,0 +1,73 @@
+package ravendb
+
+// CommandData describes data for a command
+type CommandData struct {
+	id           string
+	typ          string
```

Comment:
In Go `type` is a keyword so it cannot be used as field/variable/function name, kind of like why Java uses `clazz` instead of `class` for variable names. So I settled on using `typ` in those cases.

Reply:
How rude of `Go`